recipe/deactivate-gfortran.sh

```diff
@@ -105,6 +105,8 @@ _tc_activation \
 # extra ones:
 unset FC
 unset F95
+unset F90
+unset F77
```

Comment:
This is not the correct way. See https://github.com/conda-forge/ctng-compiler-activation-feedstock/blob/master/recipe/deactivate-gfortran.sh#L107-L112

Reply:
This was copied out of the upstream recipe I think. Will fix.